README.md

```diff
@@ -6,23 +6,20 @@
 - cmake 3.10
 - GNU GCC with C++17 (gcc-8 or higher), LLVM Clang (clang-7 or higher), or AppleClang (clang-1100.0.33.8 or higher)
 - HTSlib
-- folly (optional)
+- boost, jemalloc, gperftools (optional)
 - Python 3 (for running integration tests)
 
-All can be installed with [brew](https://brew.sh) or [linuxbrew](https://linuxbrew.sh) (does not require root)
-
 For compiling with **AppleClang**, the prerequisites can be installed as easy as:
 ```
-brew install libomp cmake make htslib boost folly
+brew install libomp cmake make htslib jemalloc gperftools
```

Comment:
what about boost, is it not needed anymore? But it's in the list of prerequisites.

Reply:
Sorry, I had made this change and forgot to commit it.